tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -235,7 +247,18 @@ declare var cfg:ISnippetConfig;
                 return scriptElement;
             }
 
-            let theScript = _createScript();
+            let theScript = _createScript(targetSrc);
+
+            if (appInsights.cr && loadFailed && domains.some(endpoint => targetSrc.indexOf(endpoint) !== -1)){
```

Comment:
The loadFailed flag is designed to mark the SDK script fetching as 100% failed and the snippet should stop doing everything, so we should not be using this flag.